DEPRECATED.md

```diff
@@ -5,6 +5,12 @@ As of release 1.3.0, Envoy will follow a
 
 The following features have been DEPRECATED and will be removed in the specified release cycle.
 
+## Version 1.7.0...
```

Comment:
Can we remove the 3 dots? I'm not sure it's worth adding another step here to switch this back at official release time. If so, can you update the release instructions? https://github.com/envoyproxy/envoy/blob/master/GOVERNANCE.md#cutting-a-release (@htuch heads up that when you do the data-plane-api work we will need to update the release instructions).

Reply:
Removed the dots. I'm not sure why I thought that would be helpful. I find it confusing because it's hard to tell from looking at this that 1.7 is in the future.  Maybe we can -- in a separate PR, add dates to the past releases, and have the release instructions include assigning a date to pending releases here once they are out.

Reply:
If you want to add some like you are doing in the release notes like "pending", or add dates below with "in progress" that's fine. I would just make a note in the release process check list that this file needs to be updated also. Can you do that as part of this PR?

Reply:
OK, done; let me know what you think.